### PR TITLE
docs: debugging

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Debugging & Troubleshooting docs.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Debugging & Troubleshooting
```
Comment (AnnaAleksandrowicz) [inline docs/debugging.md]: Moved all my changes from the branch here

File: docs/debugging.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+In this section you'll learn about using debug mode to view explicit messages in the player to troubleshoot detected problems.
+
+## Running the Player in Debug Mode
+
+To debug the player and view explicit messages in the console, you'll need to run the player in debug mode using one of the following options:
+
+**Option 1: Define a global window debug variable**: In your application, define at the top of your page the following window debug variable:
```
Comment (OrenMe) [inline docs/debugging.md]: Split to title and body, the double `:` is a bit confusing in same line.
```
**Option 1: Define a global window debug variable** 
In your application, define at the top of your page the following window debug variable:
```

File: docs/debugging.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+```js
+window.DEBUG_KALTURA_PLAYER = true;
+```
+**Option 2: Add a query string parameter to the page URL**: In your page URL, add the `debugKalturaPlayer` query string parameter:
```
Comment (OrenMe) [inline docs/debugging.md]: same line break as above

File: docs/debugging.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+```js
+http://my/page/url?debugKalturaPlayer
+```
+
```
Comment (OrenMe) [inline docs/debugging.md]: 3rd option is via player config

File: docs/debugging.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+```js
+window.DEBUG_KALTURA_PLAYER = true;
+```
+**Option 2: Add a query string parameter to the page URL**
+
+In your page URL, add the `debugKalturaPlayer` query string parameter:
+```js
+http://my/page/url?debugKalturaPlayer
+```
+**Options 3: Using Player Config**
```
Comment (AnnaAleksandrowicz) [inline docs/debugging.md]: Do we want to add info here or to send them to another page? @OrenMe ?

Comment (OrenMe) [reply docs/debugging.md]: Small note on the debugLevel would suffice here

**Pull request merged**